test/send_test.rb

```diff
@@ -40,6 +40,9 @@
             To: '+16145555555',
             From: '+16143333333',
             Body: 'This is the body'
+          },
+          headers: {
+            'User-Agent' => "Ruby Twilito #{Twilito::VERSION}"
```

Comment:
The build is failing because this test expects a space between Twilito and the version number but the code uses a space. I'm not sure which is more correct or expected, but once they match up I think this is good to merge!

Reply:
Yup, this is my fault. Changed my mind on the format after writing the tests and apparently didn't run them again. Will update.